context.go

```diff
@@ -45,15 +45,24 @@ func (c *Ctx) SourceManager() (*gps.SourceMgr, error) {
 	return gps.NewSourceManager(analyzer{}, filepath.Join(c.GOPATH, "pkg", "dep"))
 }
 
-// LoadProject searches for a project root from the provided path, then loads
-// the manifest and lock (if any) it finds there.
+// LoadProject takes a path and searches up the directory tree for
+// a project root.  If an absolute path is given, the search begins in that
+// directory.  If a relative or empty path is given, the search start is computed
+// from the current working directory.  The search stops when a file with the
+// name ManifestName (manifest.json, by default) is located.
 //
-// If the provided path is empty, it will search from the path indicated by
-// os.Getwd().
+// The Project contains the parsed manifest as well as a parsed lock file, if
+// present.  The import path is calculated as everything past Ctx.GOPATH/src.
```

Comment:
nit: do you think "the path segment below" might be a bit clearer than "everything past"?